homeassistant/util/yaml.py

```diff
@@ -156,6 +157,21 @@ def _ordered_dict(loader: SafeLineLoader,
     return processed
 
 
+def _construct_seq(loader, node):
```

Comment:
I sense some duplication here.

What does this do that [SafeLineLoader](https://github.com/persandstrom/home-assistant/blob/ae4928a2a2c6c51c78d5bef3078236c8564d553e/homeassistant/util/yaml.py#L25-L34) doesn't do?